src/qml/main.qml

```diff
@@ -106,6 +107,12 @@ Application {
                     height: DeviceInfo.hasSpeaker ? Dims.h(16) : 0
                     clip: true
                 }
+                ListItem {
+                    //% "Nightstand"
```

Comment:
Could you update the translation files so weblate picks this up ?

Reply:
Can you provide a hint as to how?  I haven't been able to figure it out yet.

Reply:
"Back in my days" 👴 this would have been a simple `lupdate asteroid-settings.pro` but now that we switched to CMake the incantation is different (and badly documented) the best I could find is https://gist.github.com/giraldeau/546ba5512a74dfe9d8ea0862d66db412 I hope it helps

Reply:
Since the move to CMake I use the following command to update translations:
```
lupdate-qt5 -no-obsolete src/* i18n/*.h -ts i18n/*.ts
```

Reply:
Done!